server/main.lua

```diff
@@ -1,15 +1,24 @@
 local QBCore = exports['qb-core']:GetCoreObject()
 local Routes = {}
 
-QBCore.Functions.CreateCallback("garbagejob:server:NewShift", function(source, cb)
+local function CanPay(Player)
+    local bank = Player.PlayerData.money['bank']
+    if bank >= Config.TruckPrice then
+        return true
+    else
+        return false
+    end
```

Comment:
```suggestion
    return Player.PlayerData.money['bank'] >= Config.TruckPrice
```
Code can be shortened here